api/models/userDao.js

```diff
@@ -0,0 +1,78 @@
+const appDataSource = require("./dataSource");
+const bcrypt = require("bcrypt")
+
+const getUser = async (userId) => {
+    const user = await appDataSource.query(`
+        SELECT id,
+               email,
+               name,
+               CONCAT(bank_name, "은행") AS bankName,
+               bank_account            AS bankAccount,
+               artist_registration     AS artistRegistration,
+               bid_agreement           AS bidAgreement,
+               CONCAT(YEAR(created_at), "년 ", MONTH(created_at), "월 ", DAY(created_at), "일 ",
+                      HOUR(created_at), "시 ", MINUTE(created_at), "분 ", SECOND(created_at),
+                      "초")             AS createdAt
+        FROM users
+        WHERE id = ?`, [userId])
+
+    if (user[0]["bidAgreement"] === 1) {
+        user[0]["bidAgreement"] = "동의"
+    } else {
+        user[0]["bidAgreement"] = "비동의"
+    }
```

Comment:
```suggestion
    const [user] = await appDataSource.query(`
        SELECT id,
               email,
               name,
               CONCAT(bank_name, "은행") AS bankName,
               bank_account            AS bankAccount,
               artist_registration     AS artistRegistration,
               bid_agreement           AS bidAgreement,
               CONCAT(YEAR(created_at), "년 ", MONTH(created_at), "월 ", DAY(created_at), "일 ",
                      HOUR(created_at), "시 ", MINUTE(created_at), "분 ", SECOND(created_at),
                      "초")             AS createdAt
        FROM users
        WHERE id = ?`, [userId])
```
if 와 else로 작성된 로직은 sql 상으로 충분히 구현 가능합니다. mysql에서 이에 관련된 구문을 찾은 뒤 적용해 수정해주세요!